ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/time-settings/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import styled from '../../styles/styled-components'
+import * as PropertyView from '../../../component/property/property.view'
+
+import MarionetteRegionContainer from '../../container/marionette-region-container'
+
+const Root = styled.div`
+  overflow: auto;
+  padding: ${props => props.theme.minimumSpacing}
+    ${props => props.theme.minimumSpacing};
+`
+
+const Time = styled.div`
+  width: 100%;
+  font-weight: bolder;
+  padding-top: 10px;
+  padding-right: 10px;
+  padding-bottom: 10px;
+  padding-left: 10px;
+`
+
+const TimeLabel = styled.div`
+  padding-top: 10px;
+  padding-bottom: 10px;
+`
+
+const TimeValue = styled.div`
+  padding-top: 10px;
+  padding-left: 10px;
```

Comment:
❓Shouldn't these use theme values like the Root padding? How does this scale when the user changes the theme spacing?